.storybook/preview-head.html

```diff
@@ -1,6 +1,8 @@
 <script>
   document.title = 'Canvas Kit Storybook';
 </script>
+<!-- Polyfill for IE11 WeakSet -->
+<script crossorigin="anonymous" src="https://polyfill.io/v3/polyfill.min.js?features=WeakMap%2CWeakSet%2CSet"></script>
```

Comment:
do we still need this since we're not supporting IE11 in v8?

Reply:
yes, it's needed until we finally deprecate IE.  It's not possible to set browser list only for specific branch but we still needs to run IE tests for v7.

Reply:
Gotcha! But this is going into `major` for some reason i thought we weren't support IE11 in `major`

Reply:
I forgot to tell that it's Chromatic tests. we should run visual tests on all browsers, because on v7 we still support it. In chromatic there is no way to exclude IE only for this branch.

Reply:
ahhh! yep yep ok thanks!

Reply:
Maybe once we release this we can add a ticket to remove it since it won't be necessary